rosetta-lang/src/main/java/com/regnosys/rosetta/formatting2/RosettaFormatter.xtend

```diff
@@ -236,6 +237,18 @@ class RosettaFormatter extends AbstractRosettaFormatter2 {
 		ele.regionFor.assignment(nameAssignment_1)
 			.prepend[oneSpace]
 	}
+	
+	def dispatch void format(Choice ele, extension IFormattableDocument document) {
+		ele.regionFor.keyword('choice')
```

Comment:
I think it is good practice to use string-based keywords as little as possible, since they are a bit error prone. E.g., if we change the keyword to be `"union"` instead of `"choice"` in the parser, we wouldn't notice that this needs changing. However, by using `choiceAccess.choiceKeyword_0` (or whatever it is), you would be notified by an error in the IDE, since `choiceKeyword_0` doesn't exist anymore.